flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -2003,19 +2005,15 @@ var A320_Neo_UpperECAM;
         }
         getEGTGaugeValue() {
             const engineId = this.index + 1;
-            const imbalance = SimVar.GetSimVarValue("L:A32NX_ENGINE_IMBALANCE", "number");
-            let egtImbalance = 0;
-            if (parseInt(imbalance.toString().substr(0, 1)) == engineId) {
-                egtImbalance = parseInt(imbalance.toString().substr(1, 2));
-            } else {
-                egtImbalance = parseInt(0);
-            }
-            const value = SimVar.GetSimVarValue("L:A32NX_ENGINE_EGT:" + engineId, "celsius") - egtImbalance;
+            const value = SimVar.GetSimVarValue("L:A32NX_ENGINE_EGT:" + engineId, "celsius");
             return value;
         }
         getN1GaugeValue() {
             const engineId = (this.index + 1);
-            const value = SimVar.GetSimVarValue("ENG N1 RPM:" + engineId, "percent");
+            const value = SimVar.GetSimVarValue("L:A32NX_ENGINE_N1:" + engineId, "percent");
+            if (value < 0.0) {
+                value = 0;
+            }
```

Comment:
This doesn't work, you'll need to use a let instead.

Reply:
Done! Thanks.